app/payments/razorpay-sdk/src/Utility.php

```diff
@@ -29,10 +29,18 @@ public function verifySignature($payload, $expectedSignature)
         // Use lang's built-in hash_equals if exists to mitigate timing attacks
         if (function_exists('hash_equals'))
         {
-            return hash_equals($actualSignature, $expectedSignature);
+            $verified = hash_equals($actualSignature, $expectedSignature);
+        }
+        else
+        {
+            $verified = $this->hashEquals($actualSignature, $expectedSignature);
```

Comment:
@riagarg5 Please take a look at this PR: https://github.com/razorpay/razorpay-php/pull/32